boards/nxp/imx95_evk/imx95_evk_mimx9596_m7.dts

```diff
@@ -23,6 +23,27 @@
 	};
 };
 
+&emdio {
+	pinctrl-0 = <&emdio_default>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	phy0: phy@1 {
+		compatible = "realtek,rtl8211f";
+		reg = <0x1>;
+		status = "okay";
+	};
+};
+
+&enetc_psi0 {
+	local-mac-address = [00 00 00 01 02 00];
+	pinctrl-0 = <&eth0_default>;
+	pinctrl-names = "default";
+	phy-handle = <&phy0>;
+	phy-connection-type = "rgmii";
+	status = "okay";
```

Comment:
I suggest to disable all related nodes by default in board dts, and enable them in dts overlay, then can use "`-DEXTRA_DTC_OVERLAY_FILE=enet1.overlay`" if needed.

Reply:
Thanks Jiafei. Will do that.